src/components/select/bl-select.ts

```diff
@@ -432,6 +432,11 @@ export default class BlSelect<ValueType extends FormValue = string> extends Form
                 ? searchbarPlaceholderText
                 : this.label || searchbarPlaceholderText}
               @input=${this._handleSearchOptions}
+              @keydown=${(e: KeyboardEvent) => {
+                if (e.code === "Space") {
+                  e.stopPropagation();
+                }
+              }}
```

Comment:
on line 576, there is a code block below. We can achieve same behaviour by updating that part of bl-select component.
```typescript
  private handleKeydown(event: KeyboardEvent) {
    if (this.focusedOptionIndex === -1 && ["Enter", "Space"].includes(event.code)) {
      this._togglePopover();
      event.preventDefault();
    }
...
```


if we don't to close popover when user press `Space`, we can achieve this behaviour by removing space in that array `(["Enter", "Space"])` in condition. Because probably we wanted to close popover our own, if now we dont want, we should update that condition I think. What do you think?

Reply:
Not closing the popover if nothing is focused makes sense to me too. @buseselvi what do you say?

Reply:
We'll discuss this further later.